### ISSUE TITLE
log: if CLI remains open for a long time, get so many sampler warn log

### ISSUE THREAD

**Issue by vagetablechicken** [4paradigm/OpenMLDB#2148]
```
W0712 10:15:22.106148 129511424 sampler.cpp:189] bvar is busy at sampling for 2 seconds!
```
brpc log, maybe we should avoid to print it on terminal too, redirect to log file.

**Comment by aceforeverd** [comment]
this seems important same as #1500 ?

**Comment by aceforeverd** [comment]
@dl239 will you kindly fix this together with #2117 ?  Both issues may have similar solution in end user side.

**Comment by aceforeverd** [comment]
another place may produce error log in cli:
https://github.com/aceforeverd/fedb/blob/25166e447b2b7185cdf589ebc18ae6a51fbc610e/hybridse/src/vm/sql_compiler.cc#L329
